reference/concepts/default_arguments.md

```diff
@@ -0,0 +1,3 @@
+A default argument is an argument to a function that a programmer is not required to specify. In most programming languages, functions may take one or more arguments. Usually, each argument must be specified in full (this is the case in the C programming language). Later languages (for example, in C++) allow the programmer to specify default arguments that always have a value, even if one is not specified when calling the function.
```

Comment:
The header followed by a newline is missing.

I would also link to argument and function in our own reference.

Reply:
Added the header and a link to functions. We don't have a reference for arguments, though.